tests/test_auto_offer_on_device_rest.py

```diff
@@ -24,9 +38,10 @@ def on_market_cycle(self, market_info):
         """
         if self.registered is False or self.is_finished is True:
             return
+        #sleep(randint(1,10)/100.)
```

Comment:
If you dont use it, please remove this line and also the import at the top.

Reply:
Sure, I will do it